package.json

```diff
@@ -11,9 +11,10 @@
   },
   "main": "main.js",
   "scripts": {
-    "start": "node main.js"
+    "start": "nodemon main.js"
```

Comment:
Why change the default script if `nodemon` is only a `devDependency`? We should rather add another script that starts `nodemon`, shouldn't we?